packages/manager/modules/core/src/translate/ovh-ui-angular/index.js

```diff
@@ -0,0 +1,110 @@
+import angular from 'angular';
+import 'angular-translate';
+import 'ovh-ui-angular';
+import '@uirouter/angularjs';
+import _ from 'lodash';
```

Comment:
Can we avoid importing full lodash?

Reply:
https://github.com/lodash/babel-plugin-lodash would be useful ;)